### ISSUE TITLE
what about some love for Buddycloud?

### ISSUE THREAD

**Issue by imaginator** [libreprojects/libreprojects#95]

**Comment by jancborchardt** [comment]
Yup, let’s do it! Can you prepare a pull request following the [guide on how to add a new project](https://github.com/libreprojects/libreprojects#adding-new-projects)?

By the way, I’m in Munich mostly now so we should meet up some time! If you’re here tomorrow, I’ll be at the [ownCloud meetup](http://www.techism.de/events/owncloud-treffen-im-werk1-coworkingspace-mach-mit-bei-owncloud-1943982/).

**Comment by imaginator** [comment]
Thanks!

Hi Jan.

Didn't know there was an owncloud meetup in Munich. I'll definitely come to
the next one (tonight I need to work on my speech for BerlinJS)

Pull requests in https://github.com/libreprojects/libreprojects/pulls

S.

On 15 April 2014 18:33, Jan-Christoph Borchardt notifications@github.comwrote:

> Yup, let’s do it! Can you prepare a pull request following the guide on
> how to add a new projecthttps://github.com/libreprojects/libreprojects#adding-new-projects
> ?
> 
> By the way, I’m in Munich mostly now so we should meet up some time! If
> you’re here tomorrow, I’ll be at the ownCloud meetuphttp://www.techism.de/events/owncloud-treffen-im-werk1-coworkingspace-mach-mit-bei-owncloud-1943982/
> .
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/libreprojects/libreprojects/issues/95#issuecomment-40503624
> .

## 

Simon Tennant | buddycloud.com | +49 17 8545 0880 | office hours:
goo.gl/tQgxP
